ui/src/HttpMF.elm

```diff
@@ -8,6 +8,15 @@ import Json.Encode as E
 import Url.Builder as B
 
 
+url =
+    { base = "http://localhost"
```

Comment:
Take care that this will have to be configurable through ENV variable to work in Docker, I think.

Reply:
Ok. Sure. We'll add a variable. We'll probably need some bash script to run the app and to set env vars or something similar.